includes/Modules/AdSense/AMP_Tag.php

```diff
@@ -92,4 +105,40 @@ private function amp_content_add_auto_ads( $content ) {
 		);
 	}
 
+	/**
+	 * Set Web Story Ad Slot ID
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param string $ad_slot_id The Ad Slot ID.
+	 */
+	public function set_story_ad_slot_id( $ad_slot_id ) {
+		$this->story_ad_slot_id = $ad_slot_id;
+	}
+
+	/**
+	 * Adds the AMP Web Story auto ads code if enabled.
+	 *
+	 * @since n.e.x.t
+	 */
+	private function render_story_auto_ads() {
+		printf(
+			'
+				<amp-story-auto-ads>
+					<script type="application/json">
+						{
+							"ad-attributes":
+							{
+								"type": "adsense",
+								"data-ad-client": "%s",
+								"data-ad-slot": "00000000"
```

Comment:
I think `00000000` should be replaced with `%s`, right?

Reply:
Yes correct, I've udpated.